modelon/impact/client/entities/workspace.py

```diff
@@ -171,6 +173,15 @@ def __repr__(self) -> str:
     def __eq__(self, obj: object) -> bool:
         return isinstance(obj, Workspace) and obj._workspace_id == self._workspace_id
 
+    @property
+    def size(self) -> float:
+        """Workspace size in bytes."""
```

Comment:
This does not feel like right interface since size of workspace may be changing during the simulation. 
Options are:
1. Always fetch the size in this property and skip constructor argument
2. Use a method `def get_size(refresh=true)` to provide caching support. In that case response from `workspace_get` call should be cached into self._size. `refresh=true` means the size is recomputed.

Reply:
Hej @iakovn ,  Good point! I have chosen option 1. :)